specs/jsonschema-core.md

```diff
@@ -415,6 +415,12 @@ keywords MUST NOT begin with this prefix.
 Implementations MUST refuse to evaluate schemas which contain keywords which
 they do not know how to process or explicitly choose not to process.
 
+## Specification Versioning and Compatibility
+
+This specification is versioned by two values: iteration and release year.
+
+A schema written to conform with the requirements of a given version (iteration and release year) is compatible with specifications published with the same iteration value and either the same or greater release year value.  Thus, JSON Schema provides a guarantee of compatibility for future releases within an iteration.
```

Comment:
Is the linter not working? It should have flagged this line as too long.

Reply:
Hm.. maybe.  I'll update it.